viewproduct.php

```diff
@@ -10,162 +10,157 @@
-                     <tr>
-                        <td><b>Review: </b><?php echo $data['Review']; ?></td>
-                     </tr>
-                     <tr>
-                        <td>
-                           <hr color="#00CCCC">
-                        </td>
-                     </tr>
-                     <?php } } else {?>
-                     <tr>
-                        <td>'0' Reviews for this product</td>
-                     </tr>
-                     <?php } ?>
-                  </table>
-                  <br>
-                  <?php if(isset($_SESSION['username'])){?>
-                  <table>
-                     <tr>
-                        <th>Leave a Review</th>
-                     </tr>
-                     <tr>
-                        <td><b>Review*</b> <textarea name="review" required></textarea></td>
-                     </tr>
-                  </table>
-                  <table>
-                     <tr>
-                        <td ><?php echo $MESSAGE_REVIEW; ?></td>
-                        <td colspan="6" class="tf"><input type="submit" name="addreview" class="cart_btn" value="Add Review"></td>
-                     </tr>
-                  </table>
-                  <?php }?>
-               </form>
+<form method="post">
+  <?php
+  $no_of_rows = mysqli_num_rows($single_product);
+  if ($no_of_rows > 0) {
+    while ($data = mysqli_fetch_array($single_product)) { ?>
+      <div class="col_3">
+        <?= '<input type="hidden" name="productid" value="' . $data['ProductID'] . '" />' ?>
+        <img src="<?= $data['ProductImage'] ?>" width="270" height="270" alt="<?= $data['ProductName'] ?>">
+      </div>
+      <div class="col_3">
+        <div class="product_description">
+          <span><?= $data['ProductBrand'] ?></span>
+          <h3><?= $data['ProductName'] ?></h3>
+          <hr>
+          <p>Design type: <?= $data['ProductDesignType'] ?></p>
+          <p>Bike type: <?= $data['ProductBikeType'] ?></p>
+          <p>Availability: <?= $data['ProductAvailability'] ?></p>
+          <p>Discount: <?= $data['ProductDiscountAmount'] ?>%</p>
+        </div>
+        <div class="product_note">Note: This item is a universal custom fit component, and is not a direct fitment for your machine. These parts may require modification and we recommend installation by a certified mechanic. Please have machine specifications ready when you call.</div>
+        <p>Quantity: <input type="number" value="1" name="quantity" min="1" step="1" class="input_quantity" required></p>
+        <div class="product_price">
+          <span>LKR:<?= $data['ProductPrice'] ?></span>
+          <a href="cart.php"><input name="addtocart" type="submit" value="Add to cart" class="cart_btn"></a>
+        </div>
+        <span><s><?= $data['InitialProductPrice'] ?></s></span>
+      </div>
+    <?php } } else {?>
+      <h3>Oops! Product Unavailable!</h3>
+  <?php } ?>
+</form>
+<form method="post">
+    <table>
```

Comment:
- Move the form tag outside the loop since it doesn't depend on the data being fetched. 
- Remove unnecessary spaces and characters between HTML tags to reduce file size and load time. 
- Use short echo syntax <?= $variable ?> instead of <?php echo $variable ?> . 
- Add alt attributes to images for accessibility